doc/src/reference.xml

```diff
@@ -561,13 +561,15 @@ path-constant DATA : data/a.txt ;
       <varlistentry>
         <term><literal>cflags</literal></term>
         <term><literal>cxxflags</literal></term>
+        <term><literal>compileflags</literal></term>
```

Comment:
AMDG

On 04/29/2016 10:41 AM, Edward Diener wrote:
>> @@ -561,13 +561,15 @@ path-constant DATA : data/a.txt ;
>>        <varlistentry>
>>          <term><literal>cflags</literal></term>
>>          <term><literal>cxxflags</literal></term>
>> +        <term><literal>compileflags</literal></term>
> 
> I do not see how writing that 'cflags' applies to both C and C++ can be correct.

Sadly it is.

In Christ,
Steven Watanabe